### ISSUE TITLE
CQL: Example 5 wrong

### ISSUE THREAD

**Issue by constantinius** [opengeospatial/ogcapi-features#554]
The JSON example for Example 5 seems wrong:

```
{
  "like": [
    { "property": "name" },
    "Smith."
  ],
  "singleChar": ".",
  "nocase": true
}
```

IMO It should be like this, as all other examples show the whole predicate node, not just the contents.

```
{
  "like": {
    "like": [
      { "property": "name" },
      "Smith."
    ],
    "singleChar": ".",
    "nocase": true
  }
}
```

**Comment by cportele** [comment]
See also #541 and #547.

**Comment by pvretano** [comment]
Resolved by PR #592.
